taskt/Core/Automation/Commands/Variable Commands/SetVariableIndexCommand.cs

```diff
@@ -1,111 +1,87 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
-using System.Data;
 using System.Collections.Generic;
 using System.Windows.Forms;
-using System.Drawing;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Variable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to modify variables.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to modify the value of variables.  You can even use variables to modify other variables.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements actions against VariableList from the scripting engine.")]
+    [Group("Variable Commands")]
+    [Description("This command sets the current index of a variable.")]
```

Comment:
This command is honestly really strange. I can't see a reason for wanting to set the index position of a variable in the engine's variable list. I feel like the user shouldn't have that kind of access to the engine.

Reply:
I think It's about setting the index of a complex variable like index of a list variable but I don't find a use of this command. For now, we can keep it.

Reply:
No if you look at the code all it's doing is changing the position of a variable in the engine VariableList, not any generic list.